publisher-sdk/src/main/java/com/criteo/publisher/CriteoNotInitializedException.java

```diff
@@ -20,7 +20,7 @@
 
   public CriteoNotInitializedException(String message) {
     super(message + "\n"
-        + "Did you initialize the Criteo SDK ?\n"
+        + "Did you properly initialize the Criteo SDK ?\n"
```

Comment:
NIT: Why asking a question as if we weren't sure? I would suggest an informative/neutral message instead.

Reply:
Because we are not really sure :) For instance, one of the publishers that reporter a problem in this area did initialize it properly.